src/openms/source/FORMAT/PepXMLFile.cpp

```diff
@@ -762,6 +763,19 @@ namespace OpenMS
         peptide_hit_.setScore(value);
         current_peptide_.setScoreType(name);
         current_peptide_.setHigherScoreBetter(false);
+        if (search_engine_ == "Comet")
+        {
+          peptide_hit_.setMetaValue("MS:1002257", value);
+        }
+        else if (search_engine_ == "X! Tandem")
+        {
+          peptide_hit_.setMetaValue("MS:1001330", value);
+        }
+        else if (search_engine_ == "Mascot")
+        {
+          peptide_hit_.setMetaValue("MS:1001172", value);
+        }
+        //TODO: there is no generic expect term in the CV right now
```

Comment:
meant was not only generic but one that is parent to all. Right now, the cv is a little fragmented on the SE scores front.